start.sh

```diff
@@ -47,22 +47,26 @@ else
     DOMAIN=$(echo "${ENV_JSON}" | jq -r -c "${DOMAIN_JQ} | values")
 fi
 
+echo "$PTVSD_ENABLE"
 echo "Waiting for DB"
-while ! nc -z ${MYSQL_HOST} ${MYSQL_PORT}; do   
+while ! nc -z "${MYSQL_HOST}" "${MYSQL_PORT}"; do   
   sleep 1 # wait 1 second before check again
 done
 
 echo "Setting Git info variables"
-export GIT_REPO="$(git config --local remote.origin.url)"
-export GIT_COMMIT="$(git rev-parse HEAD)"
-export GIT_BRANCH="$(git name-rev $GIT_COMMIT --name-only)"
+GIT_REPO="$(git config --local remote.origin.url)"
+export GIT_REPO
+GIT_COMMIT="$(git rev-parse HEAD)"
+export GIT_COMMIT
+GIT_BRANCH="$(git name-rev "$GIT_COMMIT" --name-only)"
+export GIT_BRANCH
```

Comment:
what is purpose of changing from one line to 2 liner, any benefit?

Reply:
Running shellcheck it says

`       ^------^ SC2155: Declare and assign separately to avoid masking return values.`

This was why I moved it to pass the linter.